proarc-common/src/main/java/cz/cas/lib/proarc/common/imports/ImportProcess.java

```diff
@@ -270,10 +271,15 @@ private static void deleteFolder(File folder) {
         }
     }
 
-    static List<TiffImporter> getConsumers() {
+    static List<ImageImporter> getConsumers() {
         if (consumerRegistery == null) {
-            consumerRegistery = Collections.singletonList(
-                    new TiffImporter(ImportBatchManager.getInstance()));
+            consumerRegistery = new LinkedList<ImageImporter>();
```

Comment:
> consumerRegistery = new LinkedList<ImageImporter>();
Tohle bych nahradil za unmodifiableList a Arrays.asList, aby to někoho nenapadlo měnit. Ta registrace by šla později vyměnit za service lookup.